crate_universe/src/splicing.rs

```diff
@@ -118,8 +118,10 @@ impl TryFrom<SplicingManifest> for SplicingMetadata {
             .manifests
             .into_iter()
             .map(|(path, label)| {
-                let manifest = cargo_toml::Manifest::from_path(&path)
+                let manifest_content = fs::read(&path)
```

Comment:
Can you add a comment on the significance of not using `cargo_toml::Manifest::from_path`?